plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2213,7 +2248,6 @@ void read_write::push_transactions(const read_write::push_transactions_params& p
 }
 
 void read_write::send_transaction(const read_write::send_transaction_params& params, next_function<read_write::send_transaction_results> next) {
-
    try {
       auto pretty_input = std::make_shared<packed_transaction>();
       auto resolver = make_resolver(db, abi_serializer::create_yield_function( abi_serializer_max_time ));
```

Comment:
I expected `packed_transaction` serialization to get the same treatment as `transaction_trace` and `signed_block`. Here and the other `packed_transaction` serializations.

Reply:
Should revisit after determination is made on the approach for: https://github.com/eosnetworkfoundation/product/pull/145 
In cases where expanded JSON is provided re-using a cache of retrieved ABIs during parsing of the incoming JSON would speed up the parsing of that JSON and also improve the performance of creating the transaction trace.